modules/snmp/README.md

```diff
@@ -6,219 +6,254 @@ sidebar_label: "SNMP"
 
 # SNMP device monitoring with Netdata
 
-Collects data from any SNMP device and uses the [gosnmp](https://github.com/gosnmp/gosnmp)package.
+Collects data from any SNMP device and uses the [gosnmp](https://github.com/gosnmp/gosnmp) package.
 
 It supports:
 
--   all SNMP versions: SNMPv1, SNMPv2c and SNMPv3
--   any number of SNMP devices
--   each SNMP device can be used to collect data for any number of charts
--   each chart may have any number of dimensions
--   each SNMP device may have a different update frequency
--   each SNMP device will accept one or more batches to report values (you can set `max_request_size` per SNMP server, to control the size of batches).
+- all SNMP versions: SNMPv1, SNMPv2c and SNMPv3.
+- any number of SNMP devices.
+- each SNMP device can be used to collect data for any number of charts.
+- each chart may have any number of dimensions.
+- each SNMP device may have a different update frequency.
+- each SNMP device will accept one or more batches to report values (you can set `max_request_size` per SNMP server, to
+  control the size of batches).
 
 ## Configuration
-Create a config file `/etc/netdata/go.d/snmp.conf`. Following is an example of SNMPv3 config.
+
+Edit the `go.d/snmp.conf` configuration file using `edit-config` from the
+Netdata [config directory](https://learn.netdata.cloud/docs/configure/nodes#the-netdata-config-directory), which is
+typically at `/etc/netdata`.
+
+```bash
+cd /etc/netdata # Replace this path with your Netdata config directory
+sudo ./edit-config go.d/snmp.conf
+```
+
+| Parameter                    | Default value  | Description                                                                                                      |
```

Comment:
First of all: I LOVE the table! 

Regarding document structure: 
I would place it at the end of the document, since it is a reference. We start a new section 
`### Configuration parameters` where the table lives.

Reply:
The table loves you back! 😄 

---

> I would place it at the end of the document since it is a reference

I see your point, but I think it can be hard to read/understand the examples w/o going through this table first (also having "Configuration parameters" outside of "Configuration" 🤔 ).